modules/adding-a-ca-bundle-after-upgrading.adoc

```diff
@@ -19,14 +19,43 @@ There are two ways to add a Certificate Authority (CA) bundle to {productname-sh
 
 * You can use self-signed certificates in a custom CA bundle (`odh-ca-bundle.crt`) that is separate from the cluster-wide bundle.
 
+ifdef::upstream[]
+For more information, see link:{odhdocshome}/installing-open-data-hub/#understanding-certificates_certs[Understanding certificates in {productname-short}].
+endif::[]
+
+ifdef::self-managed,cloud-service[]
+
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates].
```

Comment:
Might need to wrap line 28 with `ifndef::disconnected` so that it doesn't apply to disconnected